base/linalg/exceptions.jl

```diff
@@ -32,6 +32,15 @@ end
 struct PosDefException <: Exception
     info::BlasInt
 end
+function Base.showerror(io::IO, ex::PosDefException)
+    print(io, "PosDefException: matrix is not ")
```

Comment:
`ERROR: PosDefException: matrix is not Hermitian.` is maybe not the nicest though. Should we instead have a `CholeskyException` that print one of 
```
ERROR: CholeskyException: Cholesky factorization failed; matrix is not Hermitian.
ERROR: CholeskyException: Cholesky factorization failed; matrix is not positive definite.
```